relayer-cli/src/lib.rs

```diff
@@ -24,9 +23,11 @@
 pub mod application;
 pub mod commands;
 pub mod config;
-pub mod error;
 pub mod prelude;
 
+#[allow(missing_docs)]
```

Comment:
Yeah, one issue I have with Rust macros is I couldn't figure how to allow comments to be appear within a macro body. It might take some time to figure that out so I disabled that first.

Reply:
Maybe with the `#[doc]` attribute? https://doc.rust-lang.org/rustdoc/the-doc-attribute.html

Reply:
Thanks! This might be possible. I'll have to try it out.

Reply:
Do you want to address this before we merge this PR? I can probably take a last look tomorrow or on Monday (ie. after the next Hermes release)

Reply:
I am trying to refactor the macro code in flex-error to add this in. Should be doable by today.

Reply:
I have now managed to refactor the macros in flex-error to capture doc attributes and propagate them to the generated struct. I have added an example doc comment in `relayer_cli::error::Error`.

Btw I also notice that `cargo doc` is not working on `ibc-rs` due to a possibly new lint `rustdoc::bare_urls` being added. I have also fixed that here to make `cargo doc` builds again.